app/controllers/api/proxy_rules_controller.rb

```diff
@@ -44,7 +44,7 @@ def destroy
   private
 
   def authorize!
-    current_account.independent_mapping_rules_enabled? || raise(CanCan::AccessDenied)
+    true || raise(CanCan::AccessDenied)
   end
```

Comment:
`true or ...` is always `true`, so let's remove this method too 😄

Reply:
I noticed that raising an error `before_action` would mean this controller is no longer useful, am I wrong?

Reply:
yes, but the thing is that now it will never raise, for the lazy evaluation.
`true or raise exception` => `never raise exception`

Reply:
Of course, you're right